test/test_tamper_evident.py

```diff
@@ -73,7 +73,7 @@ def test_editing_is_detected(self):
         self.assertFalse(tef.validate())
 
     def test_oneline_file_is_detected(self):
-        filename = self.write_tamper_evident(b"Am I OK?")
+        filename = self.write_tamper_evident("Am I OK?")
```

Comment:
So, looking at it, I think the reason I changed it to unicode was that it made more sense to me that way. In particular, it's talking about writing "text", and the explicit-bytes version made assumptions about the encoding of those bytes (although I suppose only in the hashline).

I know I also had difficulties with the interfaces for `ConfigParser`, which changed between 2 and 3 in how it handled unicode and strings. Not sure if that influenced my decision or not.